### PR TITLE
Homework4: Fedor Polikarpov 

### PR DESCRIPTION
**Pull request by FedorPolikarpov**

### CODE REVIEW

File: Task 3.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Даны два списка чисел. Посчитайте, сколько различных чисел
+# содержится одновременно как в первом списке, так и во втором.
+user_lst1 = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 300, 11]
```
Comment (belovmd) [inline Task 3.py:3]: this is task for sets intersect

File: Task 4.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Даны два списка чисел. Посчитайте, сколько различных чисел
```
Comment (belovmd) [inline Task 4.py:1]: it's task for sets symmetric difference

File: Task 6.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Во входной строке записан текст. Словом считается последовательность
+# непробельных символов идущих подряд, слова разделены одним или большим
+# числом пробелов или символами конца строки. Определите, сколько различных
+# слов содержится в этом тексте.
+user_input = input().split()
```
Comment (belovmd) [inline Task 6.py]: it has one line solution with sets and split

**Review by belovmd — CHANGES_REQUESTED** [review]
please add 2, 5 tasks
please address next issues:

File: Task 3.py (hunk 1 above)
Comment (FedorPolikarpov) [reply Task 3.py:3]: rdy

File: Task 4.py (hunk 2 above)
Comment (FedorPolikarpov) [reply Task 4.py:1]: rdy

File: Task 6.py (hunk 3 above)
Comment (FedorPolikarpov) [reply Task 6.py]: rdy

File: Task 2.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    dir1[country] = cit
+
+num_req = int(input("Введите кол-во городов вы хотите проверить: "))
+lst = []
+
+for i in range(num_req):
+    lst.append(input("Введите город: "))
+
+for element in lst:
+    for country, city_name in dir1.items():
```
Comment (belovmd) [inline Task 2.py:21]: O(n*m) complexity
good idea to revert dict )

**Review by belovmd — COMMENTED** [review]
missed 5th task
6

**Review by belovmd — COMMENTED** [review]
would be re-reviewed 3/22/2020

**Review by belovmd — COMMENTED** [review]
8,5

**Pull request closed without merging**